apps/researcher/src/navigation.ts

```diff
@@ -1,6 +1,11 @@
 import {createSharedPathnamesNavigation} from 'next-intl/navigation';
 
-export const locales = ['en', 'nl'] as const;
+export enum LocaleEnum {
```

Comment:
For consideration: somehow I would expect these locale values to be in a file with a different name - I probably wouldn't look in `navigation.ts` for these values but rather in, say, a `settings.ts` or `definitions.ts` file.

Reply:
I agree, I will move it to `definitions.ts`